include/bh_python/register_histogram.hpp

```diff
@@ -30,6 +30,22 @@
 #include <tuple>
 #include <vector>
 
+namespace boost {
+namespace histogram {
+namespace detail {
+
+using _atomic_type = histogram<vector_axis_variant, storage::atomic_int64>;
+
+template <>
+struct has_operator_rdiv_impl<_atomic_type, _atomic_type> : detect_base {
+    using type = mp11::mp_false;
+};
```

Comment:
@HDembinski I'm having to work around the fact that `count` claims to have a division operator, but the implementation does not actually have one, so it reports true here then fails to compile. Same issue as the one fixed for `*` in https://github.com/boostorg/histogram/pull/328. Though `*` used to work, and `/` used to be not implemented, I believe.

Reply:
Sorry again, I am getting sloppy. I am developing a fix here:
https://github.com/boostorg/histogram/pull/329
Could you please use that version instead of 1.77?

Reply:
Sure, thanks!

Reply:
Could you sync master with develop in Boost.Histogram? I can use master, develop has been force pushed to in the past, which breaks boost-histogram if we were pointing at a commit that doesn't exist anymore and gets cleaned up.

Otherwise this works.

Reply:
Sorry, I missed that, I merged develop into master now.